docs/conceptual/using-gpu-sanitizer.md

```diff
@@ -235,9 +235,8 @@ $ rocgdb <path to application>
 
 ### Using ASan with a short HIP application
 
-Refer to the following example to use ASan with a short HIP application,
+Refer to the [example](https://github.com/Rmalavally/rocm-examples/blob/Rmalavally-patch-1/LLVM_ASAN/Using-Address-Sanitizer-with-a-Short-HIP-Application.md) to use ASan with a short HIP application.
```

Comment:
The link here does not work when I paste it into a browser.

Reply:
Thanks, Brian. The original PR has been pending approval for a long time. Let me see how I can fix this for now.